docs/config/index.md

```diff
@@ -241,6 +241,10 @@ export default defineConfig(({ command, mode }) => {
 
   Vite has a list of "allowed conditions" and will match the first condition that is in the allowed list. The default allowed conditions are: `import`, `module`, `browser`, `default`, and `production/development` based on current mode. The `resolve.conditions` config option allows specifying additional allowed conditions.
 
+  :::warning resolve subpath export
+  Property ending with "/" is not supported. It is deprecated. Please contact to package author to use [`*` subpath patterns](https://nodejs.org/api/packages.html#package-entry-points) instread.
+  :::
```

Comment:
```suggestion
  :::warning Resolving subpath exports
  Export keys ending with "/" is deprecated by Node and may not work well. Please contact to package author to use [`*` subpath patterns](https://nodejs.org/api/packages.html#package-entry-points) instead.
  :::
```

Small nitpicks to clarify that it's partially supported instead of not being supported. It still works in some cases.